### PR TITLE
Base quality adjusted alignment

### PR DESCRIPTION
**Pull request by jeizenga**
Added capability to adjust alignments for base quality
- New functions to recover the underlying probability model from a set of score parameters
- New functions to adjust scoring based on the probability model
- Options to use adjusted scores in alignment
- New traceback logic to accommodate adjusted scores

### CODE REVIEW

**Comment by adamnovak** [comment]
Once this goes in vgteam/vg#368 ought to build, right?

File: src/example_adj.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	/* This table is used to transform nucleotide letters into numbers. */
+    int8_t* nt_table = gssw_create_nt_table();
+    
+	// initialize scoring matrix for genome sequences
+	//  A  C  G  T	N (or other ambiguous code)
+	//  2 -2 -2 -2 	0	A
+	// -2  2 -2 -2 	0	C
+	// -2 -2  2 -2 	0	G
+	// -2 -2 -2  2 	0	T
+	//	0  0  0  0  0	N (or other ambiguous code)
```
Comment (adamnovak) [inline src/example_adj.c]: I suspect that this comment block is no longer accurate.

**Comment by jeizenga** [comment]
I'm pretty sure yes. The compiler was complaining about the function signatures not matching the call, and this fork has the matching functions. Compiles fine on my machine anyway.

File: src/example_adj.c (hunk 1 above)
Comment (jeizenga) [inline src/example_adj.c]: Yep. I updated it.

**Pull request merged**